src/topology/category/Compactum.lean

```diff
@@ -447,3 +448,32 @@ begin
 end
 
 end Compactum_to_CompHaus
+
+namespace Compactum
+
+def forget_factors_through_CompHaus_forget : forget ≅
```

Comment:
Should the two sides be swapped? And then call this `Compactum_to_CompHaus_comp_forget`?